### ISSUE TITLE
Accept header parsing rejects valid headers

### ISSUE THREAD

**Issue by ashokdelphia** [codingjoe/django-health-check#220]
I'm using Chrome Version 73.0.3683.86, and the accept header it makes for regular web pages is:

```
Accept: text/html,application/xhtml+xml,application/xml;q=0.9,image/webp,image/apng,*/*;q=0.8,application/signed-exchange;v=b3
```

With the new Accept header parsing in v3.10.0, the last entry is rejected with:

```
    "application/signed-exchange;v=b3" is not a valid media type
```

Apologies for not catching that in review for PR #216.

I think to properly follow the [production in RFC 7231](https://tools.ietf.org/html/rfc7231#section-5.3.2), we should allow other parameters and only care if `q` is set.

**Comment by nwolff** [comment]
Just to clarify, installing django-health-check 3.10.0 and requesting the healthcheck page returns a 500:

![2019-05-07-102102_758x251_scrot](https://user-images.githubusercontent.com/40599/57284662-eda44980-70b1-11e9-9095-7e27551e0461.png)

**Comment by ashokdelphia** [comment]
> Just to clarify, installing django-health-check 3.10.0 and requesting the healthcheck page returns a 500:

Yes. #221 should fix that.

**Comment by ashokdelphia** [comment]
This is fixed in v3.10.1
